### PR TITLE
Link contributer guidelines to readme | borrowed issue and PR templates 

### LINKED ISSUE #128: Github templates needed to helpout new contributors
Github templates are needed to help out folks who are new and want to create an issue or raise pull request. These are the following work that need to be done
- [x] Add CONTRIBUTING.md
- [ ] Add link to CONTRIBUTING.md in README.md
- [ ] Add ISSUE_TEMPLATE.md
- [ ] Add PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by a2un**
Fixes #128

Continuing PR #138  here.

One extra commit happened when I tried to sync the two repos . Couldn't squash that one.

### CODE REVIEW

File: .github/conversational/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+**Note: for support questions, please use  Discord and mailing list**. This repository's issues are reserved for feature requests and bug reports.
+
+* **I'm submitting a ...**
+  - [ ] bug report
+  - [ ] feature request
+  - [ ] support request => Please do not submit support request here, see note at the top of this template.
```
Comment (ananyo2012) [inline .github/conversational/ISSUE_TEMPLATE.md]: Remove this point

File: .github/conversational/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+**Note: for support questions, please use  Discord and mailing list**. This repository's issues are reserved for feature requests and bug reports.
```
Comment (ananyo2012) [inline .github/conversational/ISSUE_TEMPLATE.md]: Remove this line as well

File: .github/conversational/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines
```
Comment (ananyo2012) [inline .github/conversational/PULL_REQUEST_TEMPLATE.md]: We don't have a commit message guideline yet. Comment this.

File: .github/conversational/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- [ ] Unit Tests for the changes that have been added (for bug fixes / features)
```
Comment (ananyo2012) [inline .github/conversational/PULL_REQUEST_TEMPLATE.md]: We don't have unit tests for the repo as well. Instead ask to post a screenshot of your change

File: .github/conversational/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
```
Comment (ananyo2012) [inline .github/conversational/PULL_REQUEST_TEMPLATE.md]: Not sure if this line should be included. @pythonindia/pycon-2018-team Any comment on this line ?

**Comment by a2un** [comment]
@ananyo2012  updated. Could you check it out

File: .github/conversational/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (realslimshanky) [reply .github/conversational/PULL_REQUEST_TEMPLATE.md]: We can remove it. I don't know how to proceed with this message.

**Comment by a2un** [comment]
Okay updating it today

**Comment by a2un** [comment]
@realslimshanky  commented out that line

**Comment by ananyo2012** [comment]
*@realslimshanky* Breaking change would mean a new dependency added in
this case without which the blog won't render properly. A pypi package
maybe.

File: .github/conversational/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+<!-- * **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?) -->
```
Comment (ananyo2012) [inline .github/conversational/PULL_REQUEST_TEMPLATE.md]: @realslimshanky You got my point on this ?

Comment (realslimshanky) [reply .github/conversational/PULL_REQUEST_TEMPLATE.md]: It's rare but might happen. I got your point. Maybe we can use a CI to test the repo properly.

**Pull request merged**